src/components/Photos.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+import {g_bw, j_color, jg_bw, jg_bw2, jg_color, jg_rings} from "../img/index";
+
+const Photos = () => {
+  return (
+    <PhotoWrapper>
+      <img src={jg_bw} alt="jg_bw" />
```

Comment:
Probably want to have better alt text here for accessibility

Reply:
will do! fixed